web/experimental/new_playground/styles.scss

```diff
@@ -541,3 +541,14 @@ a {
     display: none;
   }
 }
+
+@media screen and (max-width: 700px) {
+  .header-gist-name {
+    display: none !important;
+  }
+}
+
+// Misc
+.hide {
```

Comment:
Consider keying off a "hidden" attribute here rather than a class. That could simplify the code in 827-860 above (since all the elements would "hide" the same way).

Reply:
I tried to make this change, but the CSS class is necessary so that the element takes up space, but is invisible.